### ISSUE TITLE
卷积神经网络模型

### ISSUE THREAD

**Issue by zixie1991** [zixie1991/face-recognition#2]
## 卷积神经网络模型

**Comment by zixie1991** [comment]
### 过拟合

* [机器学习中防止过拟合的处理方法](http://blog.csdn.net/heyongluoyao8/article/details/49429629)
* [深度网络的过拟合问题讨论](http://zongwei.leanote.com/post/深度网络的过拟合问题讨论)

**Comment by zixie1991** [comment]
### 残差网络

* [秒懂！何凯明的深度残差网络PPT是这样的|ICML2016 tutorial](http://weibo.com/ttarticle/p/show?id=2309351000224013881407195793)
* [Deep Residual Learning for Image Recognition(ResNet)论文笔记](http://blog.csdn.net/u011534057/article/details/51819968)
* [Training and investigating Residual Nets](http://torch.ch/blog/2016/02/04/resnets.html)
* [深度残差网络](http://blog.csdn.net/zackzhaoyang/article/details/51657305)
* [Deep Residual Network 深度残差网络](https://zhuanlan.zhihu.com/p/22447440?refer=ItsFuture)

**Comment by zixie1991** [comment]
### 模型压缩

* [用 TensorFlow 压缩神经网络](http://fjdu.github.io/machine/learning/2016/07/07/quantize-neural-networks-with-tensorflow.html)
* [Deep Compression阅读理解及Caffe源码修改](http://blog.csdn.net/may0324/article/details/52935869)

**Comment by zixie1991** [comment]
### 反向传播

* [关于梯度消失，梯度爆炸的问题](http://blog.csdn.net/qq_29133371/article/details/51867856)

**Comment by zixie1991** [comment]
### Center Loss

* [CenterLoss——实战&源码](https://zhuanlan.zhihu.com/p/23444100?refer=hsmyy)
* [算法工程师分析Center Loss在人脸识别中的应用](http://tech.163.com/16/1223/08/C8V4V19T00097U80.html)
* [A Discriminative Feature Learning Approach for Deep Face Recognition 原理及在caffe实验复现](http://blog.csdn.net/dongfang1984/article/details/53337195)
* [论文阅读：A Discriminative Feature Learning Approach for Deep Face Recognition](http://blog.csdn.net/u014230646/article/details/53764079)
